contrib/generate_precompile.jl

```diff
@@ -138,6 +138,10 @@ function generate_precompile_statements()
         statements = Set{String}()
         for statement in eachline(precompile_file_h)
             occursin("Main.", statement) && continue
+            # check for `#x##s66` style variable names not in quotes
+            occursin(r"#\w", statement) &&
```

Comment:
Might have been better to put around line 166 where the other workarounds are.

Reply:
Sure. Maybe a comment to clarify that this is where the workarounds live?